### DISCUSSION TITLE
Close : Transforming a poor field into a rich field.

### DISCUSSION THREAD

**Discussion by meteos77** [textbrowser/biblioteq#214] [Q&A] [answered]
Several fields are unfortunately blocked with a list of values that cannot be modified by the user.
I think of  - Originality; Condition : in the books or  State : in members : OUTSIDE USA :-(

if I take Originality, my library has only original books so the field is unused.

but origin or originality (same root) could with an enumeration database make the field very important for us.

my documents can be of origin :  Purchases ; Donations ; Temporary loans ; Permanent loans

I don't know if it's a mountain or a big mountain but it would allow without creating new fields, to propose more options with the existing.

**Comment by textbrowser** [comment]
They are combination widgets.

Originality will be made editable. It will include the available values.

**Comment by textbrowser** [comment, marked as answer]
Should have been ticket(s)!

**Reply by meteos77** [reply]
ticket created #256

**Reply by meteos77** [reply]
textbrowser turned condition and origin fields into popup fields linked to database enumeration
Many thanks

**Comment by textbrowser** [comment]
The original category doesn't include a resolution option. There are too many conversations here and I don't have the time to review all of them. No one does. :P

**Reply by meteos77** [reply]
you just prefer tickets?

**Comment by textbrowser** [comment]
No one can say that the question or request was answered. I prefer correct categories. This was a request requiring an answer. If the request cannot be marked, the category is wrong.

**Reply by textbrowser** [reply]
I know you can change the category of a discussion but that means that the discussion needs to be read again and again and again and again. See? That's wasteful.

**Comment by textbrowser** [comment]
Like when I publish videos, I put them in Show & Tell because I'm showing something and that doesn't need a resolution. People can comment but I don't need to review it. If they don't like it or are trolling, I ignore it. I don't have to review trillions of discussions to see if they're resolved. I can look for blue checks in the correct category.

**Reply by meteos77** [reply]
I will try to modify the ones that are not discussions

**Reply by meteos77** [reply]
when I add 'closed in front of the subject of the discussion, does it suit you?

**Reply by textbrowser** [reply]
No. I'm not a text browser.
